ItAcademyJava/Sprint1/Tasca06/Nivell1/Exercici2/src/s106n1e2/GenericMethods.java

```diff
@@ -0,0 +1,16 @@
+package s106n1e2;
+
+public  class GenericMethods {
+	
+	
+	
+	public <T> void genericMethod(T parameter1, T parameter2, T parameter3){

```

Comment:
You mentioned in my PR that i was using `<T,S,U>`. That was because I wanted to denote that my 3 arguments can have a different generic type.

Reply:
So it's better to use one generic type for each of the parameters to assure they are not tied to eachother, right?